packages/apps/web-viewer-test/package.json

```diff
@@ -17,8 +17,10 @@
     "@bentley/itwin-client": "^2.14.0",
     "@bentley/orbitgt-core": "^2.14.0",
     "@bentley/presentation-common": "^2.14.0",
+    "@bentley/presentation-components": "^2.7.0",
```

Comment:
all of the @bentley packages must have matching versions. Is there a reason for this difference?